client/shipping-label/views/history.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import CalypsoI18n from 'i18n-calypso';
+import Gridicon from 'components/gridicon';
+import formatDate from 'lib/utils/format-date';
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+const LabelHistory = ( { labelsHistory } ) => {
+	if ( ! labelsHistory || ! labelsHistory.length ) {
+		document.getElementById( 'woocommerce-order-label-history' ).style.display = 'none';
+		return null;
+	}
+	document.getElementById( 'woocommerce-order-label-history' ).style.display = 'block';
```

Comment:
The history metabox is not empty - it contains markup generated by `add_meta_box`. But I added CSS to hide it by default and some code to show it.

By browser compatibility I meant JavaScript's `classList.add()`, but the only browser that doesn't support it is IE7, so it turns out everything is good.